addons/mail/static/src/js/composer.js

```diff
@@ -479,16 +479,21 @@ var BasicComposer = Widget.extend(chat_mixin, {
     },
 
     send_message: function () {
+        var button = $(".o_composer_button_send");
+
         if (this.is_empty() || !this.do_check_attachment_upload()) {
             return;
         }
 
+        button.prop("disabled", "1");
         clearTimeout(this.canned_timeout);
         var self = this;
         this.preprocess_message().then(function (message) {
             self.trigger('post_message', message);
             self.clear_composer_on_send();
             self.$input.focus();
+        }, function(err, event) {
+            button.removeProp("disabled");
```

Comment:
Do you mean that the composer is recreated completely? Would it be a problem to use `always`?

Reply:
It is recreated, if I `console.log` the element, I can target it. Once the message is sent, the target doesn't work anymore